src/js/dialog.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Created by danmurray on 14/12/15.
+ */
+(function (keys) {
+    Wood.Dialog = Marionette.LayoutView.extend({
+        tagName: 'wood-dialog',
+        template: _.template('' +
+            '<div id="dialog-content-container"></div>' +
+        ''),
+        regions: {
+            dialogContentContainer: '#dialog-content-container',
+        },
+        defaults:{
+          title: 'Dialog'
+        },
+        initialize: function(options){
+          var self = this;
+          this.options = _.extend({}, this.defaults, options);
+
+          this.dialog = new BootstrapDialog({
+              type: BootstrapDialog.TYPE_PRIMARY,
+              size: BootstrapDialog.SIZE_NORMAL,
+              html: true,
+          });
+        },
+        onRender: function () {
+          // var dialogContent = this.options.dialogContent;
+          // if( dialogContent ){}
+          //   this.dialogContentContainer.show(new dialogContent.view(dialogContent.options));
+        },
+        open: function(){
+            this.dialog.setTitle(this.options.title);
+            this.dialog.setMessage(this.$el)
+            this.dialog.open();
+        },
+        close: function(){
+          this.dialog.close();
+          if(this.onClose)
+              this.onClose()
+        },
+    }, {
+      show: function (options) {
+        var widget = new Wood.Dialog(options);
```

Comment:
if you make this:

    show: function (options) {
        var widget = new this(options);
        widget.render();
        widget.open();
     }

you don't need to duplicate the logic.  The `this` in that context would be the class object that is calling show (so any sub-class).